core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -108,7 +109,11 @@ private case class Context(propertyMap: WebApiDirectives,
   def metadata(user: Option[Identity]): Map[String, JsValue] = {
     Map(
       propertyMap.method -> method.value.toLowerCase.toJson,
-      propertyMap.headers -> headers.map(h => h.lowercaseName -> h.value).toMap.toJson,
+      propertyMap.headers -> headers
+        .filter(_.name != `Timeout-Access`.name)
+        .map(h => h.lowercaseName -> h.value)
```

Comment:
Scala tip: To iterate only once you can use `collect` to do a `filter`/`map` in one step:

```scala
headers.collect { 
  case h if h.name != `Timeout-Access`.name => 
    h.lowercaseName -> h.value
}
```

Reply:
thanks for the tip - likely will be adding more exclusions - there are a few synthetic headers we should drop.